robolectric/src/test/java/org/robolectric/shadows/ShadowParcelFileDescriptorTest.java

```diff
@@ -55,6 +61,176 @@ public void testOpens() throws Exception {
     assertThat(pfd.getFileDescriptor().valid()).isTrue();
   }
 
+  @Test
+  public void testAdoptFd_transferFdSuccessfully() throws Exception {
+    assumeThat(TIPS_TESTS_NOT_SUPPOSED_TO_BE_RUN_ON_WINDOWS, File.separator, Matchers.equalTo("/"));
```

Comment:
This `assumeThat` statement doesn't need any parameter defined in test, what about extracting it to a single private method? Will it affect `assumeThat` function?